idea/api-gateway/src/rabbitmq/init-rabbitmq.ts

```diff
@@ -0,0 +1,112 @@
+import { connect, Connection, Channel } from 'amqplib';
+import { RabbitMQExchanges, RabbitMQueues } from '@gear-js/common';
+
+import config from '../config/configuration';
+import { RpcResponse } from '../json-rpc/types';
+
+let connectionAMQP: Connection;
+let mainChannelAMQP: Channel;
+
+const testBalanceServicesMap: Map<string, Channel> = new Map<string, Channel>();
+const dataStorageServicesMap: Map<string, Channel> = new Map<string, Channel>();
+const rabbitMQEventMap: Map<string, (params: any) => RpcResponse> = new Map<string, (params: any) => RpcResponse>();
```

Comment:
```suggestion
const replies: Map<string, (params: any) => RpcResponse> = new Map<string, (params: any) => RpcResponse>();
```

Reply:
It's also a meaningless name. If some people will use it in another file. Do you think it does something to understand  for what the `replies`. Also from name `replies`, not give us that is collection of data `Map`.

Reply:
This variable is responsible for a specific purpose. And I think it would be nice if its name described the purpose. I'm not sure that `Map` in the variable name is really neccessary. Anyway `Event` means nothing in that case